content/en/logs/processing/processors.md

```diff
@@ -85,6 +85,10 @@ If your logs put their dates in an attribute not in this list, use the log date
 The recognized date formats are: <a href="https://www.iso.org/iso-8601-date-and-time-format.html">ISO8601</a>, <a href="https://en.wikipedia.org/wiki/Unix_time">UNIX (the milliseconds EPOCH format)</a>, and <a href="https://www.ietf.org/rfc/rfc3164.txt">RFC3164</a>.
 </div>
 
+<div class="alert alert-info">
```

Comment:
Let's move this to the notes section below, but bolded so people recognize it. We typically reserve alerts for open/closed beta and high-level information.